libs/service-portal/core/src/lib/navigation/paths.ts

```diff
@@ -95,8 +95,8 @@ export enum ServicePortalPath {
 
   // Licenses service
   LicensesRoot = '/skirteini',
-  LicensesDriving = '/skirteini/okuskirteini',
-  LicensesDrivingDetail = '/skirteini/okuskirteini/:id',
+  LicensesDetail = '/skirteini/:type',
```

Comment:
Here we could add a slug that encompasses (`provider`, `type`)
or..
we could use what we have and add optional `provider` to [`GetGenericLicenseInput`](https://github.com/island-is/island.is/blob/f0b0789438ee4639b9f97551dc15ba03caca3251/libs/api/domains/license-service/src/lib/graphql/main.resolver.ts#L57-L60) which if provided would get us exact license.
The path would then become `/skirteini/:provider/:type`

Reply:
I'm not understanding completely, lets discuss 🤔